src/libs/actions/Policy/Policy.ts

```diff
@@ -1731,6 +1731,8 @@ function createDraftInitialWorkspace(policyOwnerEmail = '', policyName = '', pol
                 ownerAccountID: sessionAccountID,
                 isPolicyExpenseChatEnabled: true,
                 areCategoriesEnabled: true,
+                areCompanyCardsEnabled: true,
+                areExpensifyCardsEnabled: false,
```

Comment:
All of these are TEAM only, you cannot create Control policy in NewDot now. Lets be explicit about the Expensify card settings too